### PR TITLE
Add ability to filter output JSON fields

### PR DESCRIPTION
**Pull request by IwoHerka**
Adds 'record_fields' and 'exclude_record_fields' to default handlers, as
well as 'include' and 'exclude' parameters to default JSON formatter
which allow to filter-out unwanted fields from output message, such as
"levelno" or "pathname".

### CODE REVIEW

File: python_logging_rabbitmq/formatters.py [hunk 1]
```diff
@@ -27,4 +34,10 @@ def format(self, record):
         if 'exc_info' in data and data['exc_info']:
             data['exc_info'] = self.formatException(data['exc_info'])
 
+        if self.include:
+            data = {f: data[f] for f in self.include}
```
Comment (maticomp) [inline python_logging_rabbitmq/formatters.py:38]: @IwoHerka are you assuming that `KeyError` is reasonable when someone will provide invalid include dict?

File: python_logging_rabbitmq/formatters.py [hunk 2]
```diff
@@ -27,4 +34,10 @@ def format(self, record):
...
+        elif self.exclude:
+            for f in self.exclude:
+               del data[f]
```
Comment (maticomp) [inline python_logging_rabbitmq/formatters.py:41]: @IwoHerka same here

File: python_logging_rabbitmq/formatters.py (hunk 1 above)
Comment (IwoHerka) [reply python_logging_rabbitmq/formatters.py:38]: Yes, it's a matter of configuration. Configuration is not validated anywhere, I see no reason to validate it here.

Comment (maticomp) [reply python_logging_rabbitmq/formatters.py:38]: Sound. Approved.

**Pull request merged**